Backpack/src/main/java/net/skyscanner/backpack/snackbar/BpkSnackbar.kt

```diff
@@ -161,6 +161,15 @@ class BpkSnackbar private constructor(
     callback?.let(callbacks::add)
   }
 
+  fun setOnDismissed(ignoreDismissAfterAction: Boolean = true, callback: () -> Unit) =
```

Comment:
moved from `Extensions`. Using the extension for `setActions` caused a warning:
> Candidate resolution will be changed soon, please use fully qualified name to invoke the following closer candidate explicitly

Reply:
A warning in the app or in Backpack?

Reply:
in the app and in the backpack sample app - whenever it is used. i think it's because both functions now provide the same signature - i think in the past you had to explicitly specify that it was a click listener (hence the extension function for simplification probably), but this is no longer needed